static/src/javascripts/projects/common/modules/discussion/api.js

```diff
@@ -1,28 +1,27 @@
 define([
     'common/modules/user-prefs',
     'common/utils/ajax',
-    'common/utils/config',
-    'common/utils/cookies'
+    'common/utils/config'
 ], function (
     prefs,
     ajax,
-    config,
-    cookies
+    config
 ) {
 
+    var toHTTPS = function (url) {
+        return url.replace(/^http:\/\//i, 'https://');
+    };
+
     /**
      * Singleton to deal with Discussion API requests
      * @type {Object}
      */
-    var root = (document.location.protocol === 'https:')
-            ? config.page.secureDiscussionApiRoot
-            : config.page.discussionApiRoot,
-        Api = {
-            root: root,
-            // TODO get rid of discussion proxy completely when we're changed over to https
-            proxyRoot: (config.switches.discussionProxy ? (config.page.host + '/guardianapis/discussion/discussion-api') : root),
-            clientHeader: config.page.discussionApiClientHeader
-        };
+    var Api = {
+        root: config.page.discussionApiRoot,
+        proxyRoot: (config.switches.discussionProxy ? (config.page.host + '/guardianapis/discussion/discussion-api') : root),
```

Comment:
this switch has a slightly different purpose now so should change the desc https://github.com/guardian/frontend/blob/master/common/app/conf/switches/FeatureSwitches.scala#L320